### PR TITLE
Update team and owner descriptions for Big Data for new User Group category

### PR DESCRIPTION
**Pull request by erikerlandson**
This is a compantion to https://github.com/kubernetes/community/pull/3304

### CODE REVIEW

File: config/kubernetes/ug-big-data/OWNERS [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - sig-big-data-leads
+  - ug-big-data-leads
```
Comment (nikhita) [inline config/kubernetes/ug-big-data/OWNERS]: This also needs to be updated in OWNERS_ALIASES:

https://github.com/kubernetes/org/blob/0345f0fc6ad302cbb52bf072094e9416b79417f1/OWNERS_ALIASES#L28

File: config/kubernetes/ug-big-data/OWNERS [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 approvers:
-  - sig-big-data-leads
+  - ug-big-data-leads
 labels:
-  - sig/big-data
+  - usergroup/big-data
```
Comment (nikhita) [inline config/kubernetes/ug-big-data/OWNERS]: This label doesn't exist today..I'll create a PR to add it once https://github.com/kubernetes/community/pull/3304 has merged.

**Comment by nikhita** [comment]
fyi @foxish 

/hold
until https://github.com/kubernetes/org/pull/525 has merged

File: config/kubernetes/ug-big-data/OWNERS (hunk 2 above)
Comment (nikhita) [reply config/kubernetes/ug-big-data/OWNERS]: The label also needs to be `ug/big-data` to stay consistent with other label conventions.

**Comment by nikhita** [comment]
> As only subprojects own code (and that's what most of these teams reference), could we maybe consolidate to just `ug-big-data` i.e., rename `sig-big-data-misc` to `ug-big-data`?

:+1:

**Comment by justaugustus** [comment]
(The links generated for tests are broken right now: https://github.com/kubernetes/test-infra/pull/11533)

The [test failure](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/org/525/pull-org-test-all/298) shows that you're not currently a Kubernetes org member.

@erikerlandson -- could you please file a [request for membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+<your-GH-handle>)?

**Comment by erikerlandson** [comment]
@justaugustus @nikhita I have applied for membership: #540

**Comment by nikhita** [comment]
@erikerlandson Could you squash the commits? This looks good, but we will need to wait to merge this until the user group has actually formed.

(and welcome to @kubernetes :smile: :tada: )

File: config/kubernetes/ug-big-data/teams.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+teams:
+  ug-big-data:
+    description: ""
+    maintainers:
```
Comment (rlenferink) [inline config/kubernetes/ug-big-data/teams.yaml]: Per #566 can you convert to be `members` instead:
```suggestion
    members:
```

File: config/kubernetes/ug-big-data/teams.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+    - erikerlandson
```
Comment (rlenferink) [inline config/kubernetes/ug-big-data/teams.yaml]: Why are we not adding the other ug-big-data-leads (`foxish` and `liyinan926`) ?

**Comment by justaugustus** [comment]
@rlenferink / @nikhita -- Looks like this might be ready to roll. Can you `/approve`?

**Comment by nikhita** [comment]
> @rlenferink / @nikhita -- Looks like this might be ready to roll. Can you /approve?

Sorry for being a stickler :see_no_evil: but I'd prefer to approve this once steering signs off on https://github.com/kubernetes/community/pull/3304.

**Comment by nikhita** [comment]
@erikerlandson could you also squash the commits? thanks! :)

**Pull request merged**